ddl/reorg.go

```diff
@@ -180,15 +180,22 @@ type reorgInfo struct {
 
 	// StartHandle is the first handle of the adding indices table.
 	StartHandle int64
-	// EndHandle is the last handle of the adding indices table.
-	EndHandle int64
-	d         *ddlCtx
-	first     bool
+	d           *ddlCtx
+	first       bool
 }
 
-func constructDescTableScanPB(tblInfo *model.TableInfo, pbColumnInfos []*tipb.ColumnInfo) *tipb.Executor {
+// PartitionID returns the partition ID of the reorgInfo, or table ID if it's not a partition.
+func (r *reorgInfo) PartitionID() int64 {
+	ret := r.Job.ID
+	if r.Job.ReorgMeta != nil && r.Job.ReorgMeta.PartitionID != 0 {
```

Comment:
@tiancaiamao  s/`r.Job.ReorgMeta.PartitionID != 0`/ `r.Job.ReorgMeta.PartitionID != r.Job.ID` .

Reply:
@tiancaiamao You look at the [code](https://github.com/pingcap/tidb/pull/6814/files#diff-3f1b2a64850e52901ab29ea4c5742766R351), If the normal table `GetPartitionInfo` gets a failure result, what might happen ?